src/gdm/resilience/components/electric_pole.py

```diff
@@ -0,0 +1,139 @@
+from datetime import datetime
+from enum import Enum
+from typing import Annotated
+
+from infrasys import Component
+from pydantic import Field
+
+from gdm.quantities import PositiveAngle, PositiveDistance, PositiveWeight
+
```

Comment:
Lots of great information in here.  Where did it come from?  What's a good way to list sources?

Reply:
Hmm actually I learned a lot from ChatGPT but will try to find references.